### ISSUE TITLE
Jitsi not in Ubuntu 16.04 software center

### ISSUE THREAD

**Issue by wayneoutthere** [jitsi/jitsi#273]
why?? :(  

I love ubuntu.... don't you?  

Thanks!

**Comment by ibauersachs** [comment]
We have packages for Debian and Ubuntu available for download: https://download.jitsi.org/jitsi/debian/

**Comment by wayneoutthere** [comment]
good day, Ibauersachs

The "Ubuntu-way" to install software is to get it from the Ubuntu Software store. This is not only easier, but also more secure. Your software could be delivered via a Snap package, also helping to ensure that it's not messing with my system.  Do you have plans to support either (or both) method(s)? I have to make a decision about SIP software for my company and I cannot expect staff to horse around with CLI.  Thanks!

**Comment by ibauersachs** [comment]
I know all that. And the way to go would be to get Jitsi into Debian (not Ubuntu directly). This has proven to be a futile effort as we're lacking the resources and knowledge to keep up with Debian's formalities.

Your staff wouldn't have to mess with the CLI: just download the .deb, double-click and install it with the package manager. Done.

**Comment by wayneoutthere** [comment]
welll, that's funny. I've been a full time dedicated ubuntu fan boy since 2009 and I could *not* figure out how to do this easy thing you mention.  I started first here, and got lost with my options: https://jitsi.org/Main/Download

then I clicked the ubuntu logo which took me here, at which point I had a hernia and brain seizure because of the stress related to choosing the right package:
https://download.jitsi.org/jitsi/debian/

I then was relieved to see the little 'use our repository' link because I thought I would at least be able to CLI myself into success:

https://jitsi.org/Main/DebianRepository

But no go.  It seemed to say 'don't do this'.  So, then I went to a search engine and typed "HOw the heck do i get bloody jitsi working on my 16.04 ubuntu machine before I cause myself physical harm?" and it brought me to this link .... wait.  I can no longer find it! Good thing grace was with me and I found it a few hours ago.  If not, I'd be without Jitsi on my machien right now which I use for business....

And somehow you think my *staff* would be able to install it?  life was awesome when I could just go to the ubuntu software centre and search it and click 'install'.  Ubuntu is so dang huge now that it seems like this should be the very first focal point of a package - the most recent LTS from Ubuntu...

but I'm just one man without your skills.  Boy I wish I had them today!
